src/securityPolicy.c

```diff
@@ -364,6 +367,8 @@ security_policy_t policyForSignTxWitness(
 
 	if (isSigningPoolRegistrationAsOwner) {
 		DENY_UNLESS(is_valid_stake_pool_owner_path(pathSpec));
+
+		// TODO warn if account is not reasonable? see also below, we should once and for all fix the TODOs there
```

Comment:
imho it makes sense to warn here and I would just drop the TODOs below - imho there's no harm in warning the users, the policies are clear on their own and I don't see an added value in the comments, it may even be counterproductive for anybody that would like to revisit the policies in the future as it may move the focus away from arguably a lot more relevant ones

Reply:
indeed, `WARN_IF(is_too_deep(pathSpec))` should imho be DENY_IF - all other occurences of that check result in a denial, so I would align that. It's kind of weird to forbid the path to be used as change but allow spending from it